### PR TITLE
Add API and slack command to readme, for client to integrate

### PR DESCRIPTION
**Pull request by b123400**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +29,43 @@ Commands
...
+- Respond: Return an UUID that can be used in the client
+
+APIs
+--
+
+`/DeviceList` GET
+
+- Request:
+
+        { "auth_token": "1234" }
```
Comment (rickmak) [inline README.md]: Get but have this request?

I propose to use HTTP Header as Auth. Here is an example request `curl http://genius-bar.co -H "X-GENIUS-BAR-AUTH: uuid4"`

File: README.md [hunk 2]
```diff
@@ -29,3 +29,43 @@ Commands
 `/DeviceAudit [Device iD]`
 
 - Respond: Show the list of last 20 register / deregister / own actions
+
+`/AuthToken`
+
+- Respond: Return an UUID that can be used in the client
```
Comment (rickmak) [inline README.md]: Respond: Return an Auth Token that can be used in the API

File: README.md [hunk 3]
```diff
@@ -29,3 +29,43 @@ Commands
...
+
+`/DeviceEdit` POST
+
+- Request
+
+        {
+            "id": 1,
+            "serial_number": "aabbcc",
+            "auth_token": "1234"
+        }
```
Comment (rickmak) [inline README.md]: Response? The affected list of device or full list of device?

File: README.md [hunk 4]
```diff
@@ -29,3 +29,43 @@ Commands
...
+
+
+`/DeviceReg`
+
+- Request
+
+        {
+            "id": 1,
```
Comment (rickmak) [inline README.md]: The id should be a meaningful to make the README.md easy to understand.

So I would prefer the following:
```
[{
  "id": "ip6-pink",
  "serial_number": "uuidfromitune"
}]
```

Using array as enable multiple Reg in future.

**Comment by b123400** [comment]
Updated readme according to comments

**Pull request merged**